cypress/integration/nav/cluster.feature

```diff
@@ -0,0 +1,43 @@
+Feature: Cluster page
+
+  Scenario: Navigating to the cluster page
+    Given I open "home" page
+    When I click "nav-cluster"
+    Then I see "cluster" in the url
+    And I see "Cluster" in the element "kd-breadcrumbs"
```

Comment:
We could make this more human readable, one of the main aims of Cucumber, by doing:

```suggestion
    And the breadcrumbs include "Cluster"
```

Reply:
Yes, but becomes less re-usable / flexible, so it is about finding a balance